### PR TITLE
:sparkles: Document paused crons

### LINKED ISSUE #1951: :memo: Document paused crons
Document the upcoming status with crons paused on unused dev environments.

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Closes #1951

## What

* Added explanation of paused crons to `/configuration/app/cron.md`
* Added some structure to the article so it wasn't just a wall of text
* Changed some minor style things
* Did not verify the information other than paused crons
* Did not investigate the overall content of the article (whether other things should be included)
* Did not talk about the difference between `paused` and `sleepy` crons (that is not 100% resolved, but `paused` should be the user-facing name)
﻿

### CODE REVIEW

File: docs/src/configuration/app/app-reference.md [hunk 1]
```diff
@@ -605,6 +589,72 @@ crons:
...
+
+To run cron jobs in a timezone other than UTC, set the [timezone property](#top-level-properties).
+
+### Paused crons
+
+Development environments are often used for a limited time and then abandoned.
+While it's useful for environments under active development to have scheduled tasks,
+unused environments don't need to run cron jobs.
+To minimize unnecessary resource use,
+crons are paused on Development environments with no deployments in 14 days.
```
Comment (GuGuss) [inline docs/src/configuration/app/app-reference.md:616]: @CollierCZ Both Development and Staging environments (non-Production environment types).

File: docs/src/create-apps/app-reference.md [hunk 2]
```diff
@@ -697,6 +696,61 @@ The crons are queued to run after the other activity finishes.
...
+### Paused crons
+
+Development environments are often used for a limited time and then abandoned.
+While it's useful for environments under active development to have scheduled tasks,
+unused environments don't need to run cron jobs.
+To minimize unnecessary resource use,
+crons on environments with no deployments are paused.
+The following table shows how long without a deployment an environment goes before crons are paused:
+
+| Plan             | Affected environments | Time before crons paused |
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/create-apps/app-reference.md:708]: I think it's somewhat standard to leave out verbs like that in heads and places with reduced space. Definitely standard in newspaper headings.

[the remark this answers is not in the document]
Comment (trolologuy) [reply docs/src/create-apps/app-reference.md:708]: Then, it's ready to merge 🚀 ✅

File: docs/src/configuration/app/app-reference.md [hunk 3]
```diff
@@ -605,6 +589,72 @@ crons:
...
+
+Development environments with deployments within the past 14 days have crons with the status `running`.
```
[the remark this answers is not in the document]
Comment (GuGuss) [reply docs/src/configuration/app/app-reference.md]: s/4/14

**Pull request merged**